s3_management/manage.py

```diff
@@ -20,7 +20,12 @@
 
 S3 = boto3.resource('s3')
 CLIENT = boto3.client('s3')
+
+# bucket for download.pytorch.org
 BUCKET = S3.Bucket('pytorch')
+# bucket mirror just to hold index used with META CDN
+BUCKET_META_CDN = S3.Bucket('pytorch-test')
+INDEX_BUCKETS = {BUCKET, BUCKET_META_CDN}
```

Comment:
```suggestion
INDEX_BUCKETS = {
    # bucket for download.pytorch.org
    S3.Bucket('pytorch'),
    # bucket mirror just to hold index used with META CDN 
    S3.Bucket('pytorch-test')
}
```

minor nit: I think it's better to just have a singular list instead of having to go to a bunch of variables to find out what the values are.

Reply:
BUCKET variable is used as a source when creating the index and recomputing the compute_sha256